libraries/FFGAppImport/AssetImport/FetchContent.cs

```diff
@@ -92,15 +91,25 @@ public bool NeedImport()
         // Determine FFG app version
         public string fetchAppVersion()
         {
+            if (UnityEngine.Application.platform == UnityEngine.RuntimePlatform.Android) return new MoMFinder(Platform.Android).RequiredFFGVersion(); // dirty hack
             string appVersion = "";
-            if (!File.Exists(finder.location + "/resources.assets"))
+            string resourcesAssets = finder.location + "/resources.assets";
+            if (!File.Exists(resourcesAssets))
             {
-                ValkyrieDebug.Log("Could not find main assets file: " + finder.location + "/resources.assets");
+                ValkyrieDebug.Log("Could not find main assets file: " + resourcesAssets);
             }
             try
             {
                 // We assume that the version asset is in resources.assets
-                AssetsFile assetsFile = new AssetsFile(finder.location + "/resources.assets", new EndianStream(File.OpenRead(finder.location + "/resources.assets"), EndianType.BigEndian));

```

Comment:
I haven't investigated, but this change is broken, reverting so that import works.